falsepositives.go

```diff
@@ -2,12 +2,14 @@ package goaway
 
 // DefaultFalsePositives is a list of words that may wrongly trigger the DefaultProfanities
 var DefaultFalsePositives = []string{
+	"analy", // analysis, analytics
```

Comment:
To be honest, this one could probably be abused (even if the proper way to type it would be `anally`), but we'd have to add `analyz` for analyze, `analyt` for analytics and `analys` for analysis, so I guess we'll just go with `analy` as you've suggested.

Ultimately, users are free to provide their own list of false positives, so it shouldn't be too much of a big deal.